### ISSUE TITLE
Systematically use samantic HTML5 tags

### ISSUE THREAD

**Issue by KonradUdoHannes** [CorrelAid/correlaid_website#338]
Currently semantic HTML5 tags are not used systematically. I think some <article> and <section> tags are used inside the CMS, but otherwise we currently don't have tags for general page structure, in particular we don't have

- header
- main
- footer
- nav
- aside

Adding these systematically (replacing the divs that currently have these functions) can improve readability and SEO without too much effort.

For a bit of reference regarding semantic tags see the following blog post with a nice image.
https://www.semrush.com/blog/semantic-html5-guide/

**Comment by KonradUdoHannes** [comment]
I checked all the semantic tags from the list above. More careful code inspection revealed, that `header`, `footer` and `nav` where already used in the `Header` and `Footer` component.

The `main` tag was not used and was introduced in the top level `+layout.svelte`.

`aside` was not introduced as it does not seem to systematically apply to any content on our site at the moment. At least my impression was that the pages currently only contain "main" information and nothing in addition.

I also checked the source for the use of "old school" styling html tags such as `b` and `i`, which should be replaced with `strong` and `em` tags. No such tags are currently in the code base itself, but so far I did not check the content of blog posts themselves.
